### PR TITLE
View pending employee requests and add employee for a manager's specific employee request.

### PR DESCRIPTION
**Pull request by Arshpreet-Singh38**
Based on the request raised by the manager, admin functionality to view the pending requests and add employee added. The department and designation will be automatically assigned depending on the request by manager. The request status will change and employee added to login data and employee data.

### CODE REVIEW

**Review by jagdeepbaidwan — APPROVED** [review]
It looks good, but i wondering if we can combine add_employee function with add_employee_department, as it will reduce the redundancy in the project.

**Comment by Arshpreet-Singh38** [comment]
The two functions add_employee and add_employee_department have different functionalities. Unlike the add_employee, the add_employee_department automatically takes the designation, department fields. It works on the request id which was raised during manager employee request.
But surely, we can work on your suggestion. Thanks.

**Review by puneetsingh3-sysc — APPROVED** [review]
I agree with jagdeeep review that its good to combine it and reuse the code (which professor wants).

File: src/admin_dept_management.c [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+    else{
+    	printf("not connected");
+        printf("%s\n", mysql_error(conn7));	
+	}
+	oo2=mysql_init(NULL);
+	mysql_real_connect(oo2, "localhost", "root", "1234","payroll", 3306, NULL, 0);
+	char password []="1234";
+	char employee_type[30];
+	printf("Enter the employee type -Employee or Manager\n");
+	scanf("%s",employee_type);
```
Comment (varinderdhanda) [inline src/admin_dept_management.c:228]: You can add a check here to validate employee type. I can add any employee type like with "abc"

**Review by varinderdhanda — COMMENTED** [review]
I have seen your its good try to follow already given reviews. individual comments are given on some lines.

**Comment by Arshpreet-Singh38** [comment]
> 
> 
> I agree with jagdeeep review that its good to combine it and reuse the code (which professor wants).

Yeah sure,I am working on that.

**Comment by Arshpreet-Singh38** [comment]
> 
> 
> I have seen your its good try to follow already given reviews. individual comments are given on some lines.

sure, I will follow the suggestions.

**Pull request merged**